reference/shipping.v3.yml

```
@@ -164,10 +165,10 @@ paths:
                     type: object
                     properties: {}
       description: |-
-        Get shipping settings.
+        Get global shipping settings.
```

Perhaps we can make this a bit more human-readable and align with customers, which I quite like. Same below, for update.

```suggestion
        Returns the global-level shipping settings.
```

---

Updated, thanks for the suggestion